src/loggers/log-imported-batch.ts

```diff
@@ -9,21 +9,25 @@ const debug = debugLib('snyk:import-projects-script');
 export async function logImportedBatch(
   message: string,
   loggingPath: string = getLoggingPath(),
-): Promise<void> {
+): Promise<string> {
   try {
+    const fileName = `${loggingPath}/${IMPORTED_BATCHES_LOG_NAME}`
     const log = bunyan.createLogger({
       name: 'snyk:import-projects-script',
       level: 'info',
       streams: [
         {
           level: 'info',
-          path: `${loggingPath}/${IMPORTED_BATCHES_LOG_NAME}`,
+          path: fileName,
         },
       ],
     });
     debug({ message }, 'Kicked off import');
     log.info({ message }, 'Kicked off import');
+    
+    return fileName
   } catch (e) {
+    return 'could no create log file'
```

Comment:
let's not return a message instead of a filename. Return `undefined` or `null` here instead and handle this on the other end.